modules/virt-edit-boot-order-web.adoc

```diff
@@ -26,3 +26,11 @@ Edit the boot order list in the web console.
 * If you use a screen reader, press the Up Arrow key or Down Arrow key to move the item in the boot order list. Then, press the *Tab* key to drop the item in a location of your choice.
 
 . Click *Save*.
+
+[NOTE]
+====
+If the virtual machine is running, changes to the boot order list will not take effect until you restart the virtual machine.
+
+You can view pending changes by clicking *View Pending Changes* next to the pencil icon of the *Boot Order* field. A list of all pending changes, that will
```

Comment:
Instead of "next to," can you specify which side of the pencil icon it's on (right or left)? This also applies to the virt-editing-vm-web module and the virt-remove-boot-order-item-web module.

Reply:
Done